lib/views/profile_screen.dart

```diff
@@ -45,266 +53,269 @@ class ProfilePageState extends State<ProfilePage> {
 
     return Scaffold(
       backgroundColor: bgColor,
-      body: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          SizedBox(height: windowHeight * 0.05),
-          Align(
-            alignment: Alignment.centerLeft,
-            child: Padding(
-              padding: EdgeInsets.only(left: windowWidth * 0.05),
-              child: const Text(
-                'Profile',
-                style: TextStyle(fontSize: 32, fontWeight: FontWeight.bold),
+      body: SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            SizedBox(height: windowHeight * 0.05),
+            Align(
+              alignment: Alignment.centerLeft,
+              child: Padding(
+                padding: EdgeInsets.only(left: windowWidth * 0.05),
+                child: const Text(
+                  'Profile',
+                  style: TextStyle(fontSize: 32, fontWeight: FontWeight.bold),
+                ),
               ),
             ),
-          ),
-
-          // User Profile Row
-          Padding(
-            padding: const EdgeInsets.only(
-                top: 10.0, bottom: 10.0, left: 20.0, right: 20.0),
-            child: Container(
-              height: windowHeight * 0.1,
+        
+            // User Profile Row
+            Padding(
+              padding: const EdgeInsets.only(
+                  top: 10.0, bottom: 10.0, left: 20.0, right: 20.0),
+              child: Container(
+                height: windowHeight * 0.1,
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(20),
+                ),
+                child: Center(
+                  child: ListTile(
+                    leading: CircleAvatar(
+                      radius: 30,
+                      backgroundImage: NetworkImage(
+                        profileData.profilePictureUrl ?? ""),
```

Comment:
We should have a default NetworkImage for when the user does not have a profile picture. The network image before should work.

Reply:
I have the UserData class set up so that the profile picburl defaults to that image

Reply:
hmm is there a need for the "??" then?

Reply:
Oh yeah you're right, I should have it as the default url anyway

Reply:
Fixed in f188e8a